clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperations.java

```diff
@@ -87,4 +87,15 @@
 
   // Image operations
   public static final String DEREGISTER_IMAGE = "deleteImage";
+
+  // AWS Lambda Operations
+
+  public static final String INVOKE_AWSLAMBDA = "invokeAwsLambda";
+  public static final String CREATE_AWSLAMBDA = "createAwsLambda";
+  public static final String UPDATE_AWSLAMBDA_CONFIGURATION = "updateAwsLambdaConfiguration";
+  public static final String UPDATE_AWSLAMBDA_CODE = "updateAwsLambdaCode";
+  public static final String DELETE_AWSLAMBDA = "deleteAwsLambda";
+  public static final String UPSERT_AWSLAMBDA_ALIAS = "upsertAwsLambdaAlias";
+  public static final String UPSERT_AWSLAMBDA_EVENTMAPPING = "upsertAwsLambdaEventMapping";
+  public static final String DELETE_AWSLAMBDA_EVENTMAPPING = "deleteAwsLambdaEventMapping";
```

Comment:
Do we have any appetite for making these cloud agnostic? E.g. `invokeFunction`, `createFunction`, and so-on.

@spinnaker/reviewers

Reply:
I was hoping to see a design doc before this PR dropped -- could you write something up @anshrma? https://www.spinnaker.io/community/contributing/submitting/#before-you-begin

Reply:
Making the operation names provider-agnostic makes sense to me. Resolved is done in the context of a given provider when executed, so don't see any downside.